data/networks.json

```diff
@@ -322,6 +326,22 @@
       {
         "url": "wss://zetachain-mainnet.public.blastapi.io",
         "type": "evm-ws"
+      },
+      {
+        "url": "https://zetachain-rpc.lavenderfive.com:443",
+        "type": "tendermint-rpc"
+      },
+      {
+        "url": "https://zetachain-api.lavenderfive.com:443",
+        "type": "cosmos-http"
+      },
+      {
+        "url": "https://testnet-zetachain-grpc.lavenderfive.com:443",
```

Comment:
Can you, please, remove what appears to be a testnet endpoint from ZetaChain's mainnet?

Reply:
Fixed!